recipes/spyrmsd/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "spyrmsd" %}
+{% set version = 0.3.0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 985861283513fcf9979c028ccd4a9c8dfb7771673a77a8182104de618950e261
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - graph_tool
+    - qcelemental
+    - openbabel
+
+test:
+  imports:
+    - spyrmsd
+
+  requires:
```

Comment:
If you aren't running any tests with `pytest`, you should remove the `requires` section from `test`.  However since `setup_requires` in your `setup.py` includes `pytest_runner`, you should include that in `host`, I believe.

Reply:
I'll check that. Tests require optional packages that I don't want for force the user to install. I'll possibly remove `pytest_runner` from `setup_requires`.

Reply:
I removed `pytest_runner` from `setup.py`; the package now only checks that `spyrmsd` can be imported (test are ensured by CI and protected `master` branch).